articles.tex

```diff
@@ -335,7 +335,14 @@
 \end{enumerate}
 
 \asection{Qualifications}
-\asubsection{Qualifications to be the Chairman, Evaluations Director, Social Director(s), Financial Director, Research and Development Director(s), House Improvements Director, House History Director}
+\asubsection{Qualifications to be the Chairman, Evaluations Director}
+\begin{enumerate}
+	\item Candidates must be Active Members during the term of office.
+	\item Candidates must reside On-Floor during the term of office.

```

Comment:
Elsewhere in the constitution, the capitalized and hyphenated "On-Floor" is used to refer to the capacity to live on floor. I would just s/On-Floor/on floor/ here, to be consistent.